lib/chef/knife/bootstrap.rb

```diff
@@ -1094,6 +1112,18 @@ def session_timeout
 
         timeout.to_i
       end
+
+      def bootstrap_version_gt_15?
+        !!config[:bootstrap_version] &&
+          (config[:bootstrap_version] == "latest" ||
+           config[:bootstrap_version].split(".").first.to_i >= CHEF_15)
+      end
+
+      def target_node_gt_15?
+        if Chef::VERSION.split(".").first.to_i < CHEF_15
```

Comment:
This should just be `if Chef::VERSION < 15.0`

Reply:
@lamont-granquist  already did in PR https://github.com/chef/chef/pull/8782